normandy/recipes/static/css/arguments_editor.css

```diff
@@ -39,3 +39,11 @@
 .arguments-editor .tabs span {
     margin-left: 5px;
 }
+
+/* The JSON editor modal hangs off the button, which is floated right.
+   It's absolutely position, so we set it's right edge to the button's
+   right edge to be able to see it. */
+.arguments-editor .modal {
+    left: unset !important;
+    right: 0 !important;
+}
```

Comment:
These are set as style attributes directly on the DOM element by JSONEditor, and they're set outside of the theme system where we can override it. That's why I'm using `!important` to override them. I'm not very proud of this.